src/ducks/Signals/chart/SignalPreview.js

```diff
@@ -59,31 +45,25 @@ const SignalPreviewChart = ({
 
   const _metrics = metrics.filter(metric => initialMetrics.includes(metric))
 
-  const amountOfTriggers = points.reduce(
-    (acc, val) => (acc += +val['triggered?']),
-    0
-  )
-
-  const { label, value } = getTimeRangeForChart(type)
-
   return (
-    <div className={styles.preview} style={{ minHeight: height }}>
+    <div className={styles.preview}>
       <div className={styles.description}>
         <span className={styles.fired}>Signal was fired:</span>{' '}
         <span className={styles.times}>
-          {amountOfTriggers} times in {label}
+          {triggeredSignals.length} times in {label}
```

Comment:
Can TriggeredSignals be undefined? @DmitriVanGuard

Reply:
[It's an array of `0..n` elements](https://github.com/santiment/app/pull/455/files/54475acc1267acab4b5c1414908f6738ea7a6477#diff-a1bbf1ac2ff9ceae22542736124f6e47R112). TypeScript would really help to avoid uncertainties in such cases